### DISCUSSION TITLE
Need to rearrange favourites list

### DISCUSSION THREAD

**Discussion by cormacrelf** [community/community#3738] [Mobile]
Currently the favourited repos are sorted by name. I work for an organisation that starts with Z. Its repos are at the bottom, and there is no way to move them to the top of the list. So the favourites are really annoying for me. I imagine anyone with more than three or four favourites would have a similar problem. Please make the list re-orderable.

**Comment by brianlovin** [comment]
Thanks for the feedback, I agree this would be a great thing to support – we don't have it on the roadmap right now, but I hope we can work on this soon.
